### PR TITLE
Woloxapp contacts

### PR DESCRIPTION
**Pull request by ignaciosantise**
## Summary
* Added contact tab
* Added search bar

## Screenshots
iOS
contacts ios

Android
contacts android

### CODE REVIEW

File: src/components/ContactsView/ContactsView.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <SearchBar
+          ref={(c) => { this.searchBar = c; }}
+          onChangeText={this.handleSearchChange}
+          onSearchButtonPress={this.handleSearchButtonPress}
+          onCancelButtonPress={()=> this.setState({filterValue:this.emptyString})}
```
Comment (mvbattan) [inline src/components/ContactsView/ContactsView.js]: spaces

File: src/components/SearchBar/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  autoCorrect: false,
+  autoCapitalize: "none",
+  clearButtonMode: "always"
+}
+
+SearchBar.propTypes = {
+  onChangeText: func.isRequired,
+  placeholder: string,
+  autoCorrect: bool,
+  autoCapitalize: oneOf(['none', 'sentences', 'words', 'characters']),
```
Comment (mvbattan) [inline src/components/SearchBar/SearchBar.js]: isRequired ?

File: src/components/SearchBar/SearchBar.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  clearButtonMode: oneOf(['never', 'while-editing', 'unless-editing', 'always'])
```
Comment (mvbattan) [inline src/components/SearchBar/SearchBar.js]: isRequired ?

File: src/components/SearchBar/SearchBar.js (hunk 2 above)
Comment (ignaciosantise) [inline src/components/SearchBar/SearchBar.js]: no, these props are optional. i've added them to make the searchbar more customizable.

if a prop is not passed, the component uses the default value

File: src/components/SearchBar/SearchBar.js (hunk 3 above)
Comment (ignaciosantise) [inline src/components/SearchBar/SearchBar.js]: same here

**Pull request merged**